src/build/build_implicitconstraints.jl

```diff
@@ -157,14 +223,14 @@ function build_implicitconstraints!(m, jm)
                     $(output.quantity) *
                     $(y_over_y_bar(jm, s)) *
                         (
-                            $(get_jump_variable_for_commodity(jm, output.commodity)) /
-                            $(create_rev_expr(jm, s)) /
-                            $(get_commodity_benchmark(output.commodity))
+                                                       $(get_jump_variable_for_commodity(jm, output.commodity)) /
+                           ( $(create_rev_expr(jm, s)) *
+                            
+                            $(get_commodity_benchmark(output.commodity)))
```

Comment:
This is the only substantial change, really? Replacing the division with a multiplication? Was that just the bug why things didn't work before?

Reply:
No, this isn't actually a change, just re-order of operations for ease of reading/interpretation.